include/dlt/dlt_common.h

```diff
@@ -83,7 +83,7 @@
 #include <time.h>
 #endif
 
-#if !defined (__WIN32__) && !defined(_MSC_VER)
+#if defined (DLT_SERIAL)
 #include <termios.h>
```

Comment:
Why __WIN32__ and __MSC_VER  is replaced by DLT_SERIAL in many places? what is the relation between these ?

Reply:
as far as I understood the source code, WIN32 ifdefs in those places was an attempt to commented out serial support which is not available on Windows.
I did it in a generic way by introducing DLT_SERIAL instead